test.config.js

```diff
@@ -3,7 +3,6 @@ require("@babel/register")({
 })
 
 require("coffeescript/register")
-require("@babel/polyfill")
```

Comment:
Why remove this?

Reply:
Jest running was displaying many errors related to this:
<img width="746" alt="Screen Shot 2019-04-11 at 5 06 32 PM" src="https://user-images.githubusercontent.com/1497424/55993086-336d1e00-5c7c-11e9-8682-5ec64e1d0674.png">

Reply:
Ahh yes, the new `preset-env` setting: https://github.com/artsy/force/blob/master/.babelrc#L9. We're already setup so 👍